lib/components/payment-button-front-end-component.js

```diff
@@ -0,0 +1,24 @@
+/** @format */
+
+import { By as by, until } from 'selenium-webdriver';
+import * as driverHelper from '../driver-helper.js';
+
+import BaseContainer from '../base-container.js';
+
+export default class PaymentButtonFrontEndComponent extends BaseContainer {
+	constructor( driver ) {
+		super( driver, by.css( '.jetpack-simple-payments-wrapper' ) );
+	}
+
+	async clickPaymentButton() {
+		const payPalPayButtonSelector = by.css( '.paypal-button' );
+		this.driver.wait(
+			until.ableToSwitchToFrame( by.css( '.jetpack-simple-payments-button iframe' ) ),
```

Comment:
I propose to use `.xcomponent-component-frame` selector instead `.jetpack-simple-payments-button iframe` and it should be fine.